### PR TITLE
Add landing message. #1

### PR DESCRIPTION
**Pull request by grigory-ungurian**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-  
-</body>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Welcom to Potionfy</title>
+  </head>
+  <body>
+    <h1>Welcome to Potionfy! Enjoy your stay.</h1>
```
Comment (alex-ungurian) [inline index.html]: Could you remove the 'Enjoy your stay.' string and make the 'Welcome to Potionfy!' string red?

Comment (grigory-ungurian) [reply index.html]: sure, can do!

**Review by alex-ungurian — CHANGES_REQUESTED** [review]
Please, change the code as per my comments on the review.

**Review by alex-ungurian — APPROVED** [review]
Well done Harry!

**Pull request merged**